### PR TITLE
Print archvive infos based on the xcarchive model

### PR DESCRIPTION
**Pull request by kdobmayer**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -707,184 +651,81 @@ is available in the $BITRISE_XCODE_RAW_RESULT_TEXT_PATH environment variable`)
...
 				for _, profileInfo := range profs {
-					printProfileInfo(profileInfo, certs)
-					fmt.Println()
+					log.Debugf(profileInfo.String(certs...))
 				}
 
 				log.Printf("Resolving CodeSignGroups...")
-				codeSignGroups := utils.ResolveCodeSignGroupItems(bundleIDs, exportoptions.Method(exportMethod), profs, certs)
+				codeSignGroups := export.CreateSelectableCodeSignGroups(certs, profs, bundleIDs)
 				if err != nil {
```
Comment (godrei) [inline main.go]: export.CreateSelectableCodeSignGroups does not produce any error

File: main.go [hunk 2]
```diff
@@ -707,184 +651,81 @@ is available in the $BITRISE_XCODE_RAW_RESULT_TEXT_PATH environment variable`)
...
-						} else {
-							log.Warnf("removing CodeSignGroup: %s", group.Certificate.CommonName)
-							fmt.Println()
+				codeSignGroups = export.FilterSelectableCodeSignGroups(codeSignGroups, filters...)
+
+				defaultProfileURL := os.Getenv("BITRISE_DEFAULT_PROVISION_URL")
+				if configs.TeamID == "" && defaultProfileURL != "" {
+					if defaultProfile, err := utils.GetDefaultProvisioningProfile(); err == nil {
+						fmt.Println()
+						log.Donef("default profile: %v", defaultProfile)
```
Comment (godrei) [inline main.go]: log.Donef -> log.Debug new lines also should be printed if debug

File: main.go [hunk 3]
```diff
@@ -707,184 +651,81 @@ is available in the $BITRISE_XCODE_RAW_RESULT_TEXT_PATH environment variable`)
...
-							}
+				iosCodeSignGroups := export.CreateIosCodeSignGroups(codeSignGroups)
 
-							if len(filteredGroups) > 0 {
-								codeSignGroups = filteredGroups
-							}
-						}
-					}
-				}
+				if len(iosCodeSignGroups) > 0 {
```
Comment (godrei) [inline main.go]: please print an error in the else case

File: main.go [hunk 4]
```diff
@@ -287,6 +247,10 @@ func main() {
 		fail("Issue with input: %s", err)
 	}
 
+	if configs.VerboseLog == "yes" {
```
Comment (trapacska) [inline main.go]: -> `log.SetEnableDebugLog(configs.VerboseLog == "yes")`

File: step.yml [hunk 5]
```diff
@@ -277,6 +277,14 @@ inputs:
       - "yes"
       - "no"
       is_required: true
+  - verbose_log: "yes"
```
Comment (trapacska) [inline step.yml]: Please set a debug category for this new input: `category: Debug`

**Pull request merged**